doc/index.rst

```diff
@@ -76,21 +76,25 @@ started on khmer development <dev/getting-started>`.
 Details
 =======
 
-:Authors: Michael R. Crusoe, Hussien F. Alameldin, Sherine Awad, Elmar
-        Bucher, Adam Caldwell, Reed Cartwright, Amanda Charbonneau, Bede
-        Constantinides, Greg Edvenson, Scott Fay, Jacob Fenton, Thomas Fenzl,
-        Jordan Fish, Leonor Garcia-Gutierrez, Phillip Garland, Jonathan Gluck,
-        Iván González, Sarah Guermond, Jiarong Guo, Aditi Gupta, Joshua R.
-        Herr, Adina Howe, Alex Hyer, Andreas Härpfer, Luiz Irber, Rhys Kidd,
-        David Lin, Justin Lippi, Tamer Mansour, Pamela McA'Nulty, Eric
-        McDonald, Jessica Mizzi, Kevin D. Murray, Joshua R. Nahum, Kaben
-        Nanlohy, Alexander Johan Nederbragt, Humberto Ortiz-Zuazaga, Jeramia
-        Ory, Jason Pell, Charles Pepe-Ranney, Zachary N Russ, Erich Schwarz,
-        Camille Scott, Josiah Seaman, Scott Sievert, Jared Simpson, Connor T.
-        Skennerton, James Spencer, Ramakrishnan Srinivasan, Daniel Standage,
-        James A. Stapleton, Joe Stein, Susan R Steinman, Benjamin Taylor, Will
-        Trimble, Heather L. Wiencko, Michael Wright, Brian Wyss, Qingpeng
-        Zhang, en zyme, C. Titus Brown
+:Authors: Daniel Standage, Russell Neches, Hussien F. Alameldin, Ali Aliyari,
```

Comment:
@standage just double checking: having Russel Neches out of alphabetical order is on purpose?